### PR TITLE
Release bigtable 0.31.1

### PR DESCRIPTION
**Pull request by sduskis**
This pull request was generated using releasetool.

### CODE REVIEW

File: bigtable/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.31.1
+
+11-02-2018 08:13 PDT
+
+### Implementation Changes
+- Bigtable: fix instance IAM methods ([#6343](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6343))
```
Comment (tseaver) [inline bigtable/CHANGELOG.md]: This is a testing-only change (fixes snippets for IAM methods).

Comment (tseaver) [reply bigtable/CHANGELOG.md]: Also:  please remove the `Bigtable:` prefixes from all entries.

File: bigtable/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,32 @@
...
+- Bigtable: Support DirectRow without a table ([#6336](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6336))
+- Bigtable: Add retry parameter to 'Table.read_rows()'. ([#6281](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6281))
+- Bigtable: Fix ConditionalRow interaction with check_and_mutate_row ([#6296](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6296))
+- Bigtable: deprecate 'channel' arg to 'Client' ([#6279](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6279))
+
+### New Features
```
Comment (tseaver) [inline bigtable/CHANGELOG.md]: Delete empty section.

File: bigtable/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,32 @@
...
+
+### Dependencies
+- Update IAM version in dependencies ([#6362](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6362))
+
+### Documentation
+- Bigtable: add 'docs/snippets.py' and test ([#6012](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6012))
+
+### Internal / Testing Changes
+- Bigtable: fix flaky systests / snippets ([#6367](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6367))
+- Add explicit coverage for 'row_data._retry_read_rows_exception'. ([#6364](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6364))
```
Comment (tseaver) [inline bigtable/CHANGELOG.md]: Markdown fix:

```suggestion
- Add explicit coverage for `row_data._retry_read_rows_exception`. ([#6364](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6364))
```

**Comment by sduskis** [comment]
@tseaver, I addressed the comments.  PTAL.

**Comment by sduskis** [comment]
Please don't merge until #6385 is merged.

File: bigtable/CHANGELOG.md [hunk 4]
```diff
@@ -16,7 +40,7 @@
 - Prepare docs for repo split. ([#6014](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6014))
 
 ### Internal / Testing Changes
-- Refactor 'read_row' to call 'read_rows' ([#6137](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6102))
+- Refactor `read_row` to call `read_rows' ([#6137](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6102))
```
Comment (tseaver) [inline bigtable/CHANGELOG.md]: Needs one more backtick:
```suggestion
- Refactor `read_row` to call `read_rows` ([#6137](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6102))
```

**Comment by tseaver** [comment]
@sduskis I added entries for the PRs landed this week.

File: bigtable/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,33 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.31.1
+
+11-02-2018 08:13 PDT
+
+### Implementation Changes
```
Comment (tseaver) [inline bigtable/CHANGELOG.md]:
```suggestion
### Implementation Changes
- Fix anonymous usage under Bigtable emulator ([#6385](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6385))
```

**Pull request merged**